### PR TITLE
Update TPM setup instructions

### PR DESCRIPTION
**Pull request by SlavyMihov**
Update TPM setup instructions.

### CODE REVIEW

File: en-US/win10/SetupTPM.md [hunk 1]
```diff
@@ -136,22 +136,22 @@ Firmware TPM (fTPM) requires special Processor/SoC support and whence fTPM is no
...
-1. Get a discrete TPM module and attach it to the MBM.
-2. Based on your discrete TPM module of choice, identify its matching ACPI table [here][3].
-3. Enable testsigning on the device:
-
-        bcdedit /set {current} integrityservices disable
-        bcdedit /set testsigning on
-
-4. Copy the ACPI table to MBM _C:\Windows\System32\ACPITABL.dat_.
-5. Disable fTPM by changing the following UEFI settings:
+1. Get a discrete TPM module and attach it to the MBM/RPi2.
```
Comment (ooeygui) [inline en-US/win10/SetupTPM.md:139]: Do you have a pointer to a supported module on the supported hardware site?

File: en-US/win10/SetupTPM.md [hunk 2]
```diff
@@ -136,22 +136,22 @@ Firmware TPM (fTPM) requires special Processor/SoC support and whence fTPM is no
...
+2. (Applies to MBM) Disable fTPM by changing the following UEFI settings:
 
         Device Manager -> System Setup -> Security Configuration -> PTT = <Disable>
 
-6. Enable dTPM by changing the following UEFI settings:
+3. (Applies to MBM) Enable dTPM by changing the following UEFI settings:
```
Comment (ooeygui) [inline en-US/win10/SetupTPM.md:144]: Can you explain this further? I don't know how to get into UEFI from here; customers also are unlikely to as well.

File: en-US/win10/SetupTPM.md [hunk 3]
```diff
@@ -136,22 +136,22 @@ Firmware TPM (fTPM) requires special Processor/SoC support and whence fTPM is no
...
 
+4. Based on your discrete TPM module of choice, identify its matching ACPI table [here][3].
+5. Copy that ACPI table to MBM/RPi2 _C:\Windows\System32\ACPITABL.dat_.
+6. Enable testsigning on the device:
+
+        bcdedit /set {current} integrityservices disable
+        bcdedit /set testsigning on
+
 7. Reboot the device.
 8. Verify you have the right TPM version enabled - run the [TPM 2.0 Tool][1] on the Windows IoT Core device.
```
Comment (ooeygui) [inline en-US/win10/SetupTPM.md:156]: The footnote link model isn't recommended. you should link it normally using markdown syntax.

File: en-US/win10/SetupTPM.md [hunk 4]
```diff
@@ -287,7 +287,7 @@ Note that **sTPM is intended for development purposes only and does not provide
         bcdedit /set {current} integrityservices disable
         bcdedit /set testsigning on
 
-4. Copy the matching ACPI table from [here][3] to RPi2 _C:\Windows\System32\ACPITABL.dat_.
+4. Copy the ACPI table from [here][4] to MBM/RPi2 _C:\Windows\System32\ACPITABL.dat_.
```
Comment (ooeygui) [inline en-US/win10/SetupTPM.md:290]: What does this mean?

File: en-US/win10/SetupTPM.md (hunk 1 above)
Comment (SlavyMihov) [inline en-US/win10/SetupTPM.md:139]: No, I do not have a pointer to supported modules that is available publicly.
The handful of TPM modules that we verified on MBM and RPi2 are not (yet) retail sold (off-the-shelf). My understanding is that the makers/OEMs need to contact the TPM Manufacturers and arrange purchase directly. This is more of a Maker-pro scenario.
merzink should be able to confirm and provide more details here.

File: en-US/win10/SetupTPM.md (hunk 2 above)
Comment (SlavyMihov) [inline en-US/win10/SetupTPM.md:144]: Getting into MBM's UEFI requires:
1. Hook up a display and a keyboard
2. Power on the MBM
3. Hit the F2 button on the keyboard when you see the prompt on the screen.
4. Voila, you are in the UEFI.
I did not feel the need of going into that much details. The TPM scenario is targeting advanced audience. Please, let me know if you feel I should elaborate on the instructions.

File: en-US/win10/SetupTPM.md (hunk 3 above)
Comment (SlavyMihov) [inline en-US/win10/SetupTPM.md:156]: Is it the hyperlink format of step 4 (for example) that does not comply?
Can you provide me an example of the correct way to do that? Alternatively I could peek somewhere/anywhere else on WoD to look up how hyperlinks are done...

File: en-US/win10/SetupTPM.md (hunk 4 above)
Comment (SlavyMihov) [inline en-US/win10/SetupTPM.md:290]: Is step 4 the one that needs to be clarified?
Basically, the user needs to copy the ACPITABL.dat file published on GitHub to the filesystem of the Athens device. The location on the Athens' filesystem (SD card) that this file should go is: C:\Windows\System32

File: en-US/win10/SetupTPM.md (hunk 1 above)
Comment (SlavyMihov) [inline en-US/win10/SetupTPM.md:139]: List of TPM manufacturers (and links to their webpages) is provided in a table in  /en-US/win10/TPM.md:
Infineon 
NationZ 
Nuvoton 
STMicro

**Pull request merged**